### PR TITLE
KYLIN-3597 Improve code smell

### PR DESCRIPTION
**Pull request by hit-lacus**

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by hit-lacus** [comment]
Local CI test pass.

File: query/src/main/java/org/apache/kylin/query/relnode/visitor/TupleFilterVisitor.java [hunk 1]
```diff
@@ -130,10 +130,10 @@ public TupleFilter visitCall(RexCall call) {
                 // is a trivial expr
                 return f;
             }
-            //else go to default
-        default:
             filter = new UnsupportedTupleFilter(TupleFilter.FilterOperatorEnum.UNSUPPORTED);
             break;
+        default:
```
Comment (shaofengshi) [inline query/src/main/java/org/apache/kylin/query/relnode/visitor/TupleFilterVisitor.java]: Here has duplicated UnsupportedTupleFilter?

File: query/src/main/java/org/apache/kylin/query/util/QueryUtil.java [hunk 2]
```diff
@@ -178,8 +178,9 @@ public static boolean isSelectStatement(String sql) {
         String sql1 = sql.toLowerCase(Locale.ROOT);
         sql1 = removeCommentInSql(sql1);
         sql1 = sql1.trim();
-        return sql1.startsWith("select") || (sql1.startsWith("with") && sql1.contains("select"))
-                || (sql1.startsWith("explain") && sql1.contains("select"));
+        String select = "select";
```
Comment (shaofengshi) [inline query/src/main/java/org/apache/kylin/query/util/QueryUtil.java]: "select" be declared as a constant?

File: query/src/main/java/org/apache/kylin/query/relnode/visitor/TupleFilterVisitor.java (hunk 1 above)
Comment (shaofengshi) [reply query/src/main/java/org/apache/kylin/query/relnode/visitor/TupleFilterVisitor.java]: I see it now; No question now.

**Pull request merged**